python-sdk/src/astro/options.py

```diff
@@ -65,3 +74,8 @@ class SnowflakeLoadOptions(LoadOptions):
 
     def empty(self):
         return not self.file_options and not self.copy_options
+
+
+@attr.define
+class WASBLocationLoadOptions(LoadOptions):
+    storage_account: str = attr.field(default=None)
```

Comment:
Shouldn't `storage_account` be part of the `SnowflakeOptions`, since it's not required by WASB in an isolated manner - one when the transfer is between any cloud storage and Snowflake?

Reply:
storage_account is specific to azure.

Reply:
storage_account is specific to azure.